themes/SuiteP/css/forms.scss

```diff
@@ -299,6 +474,150 @@ fieldset[disabled] .btn-primary.active {
   border-radius: 4px;
 }
 
+#actionLines {
+  display: block;
+}
+
+#actionLines > tr {
+  display: block;
+}
+
+#actionLines > tr > td {
+  display: block;
+}
+
+#actionLines td[field] label {
+  margin-right: 8px;
+}
+
+#actionLines > tr > td {
+  border: 1px solid $workflow-action-bg;
+  border-top-left-radius: 4px;
```

Comment:
i mean do you need to use border-top-left-radius, border-top-right-radius etc when they are all the same just use border-radius: 4px;